src/components/Login.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div
+    class="uk-height-1-1 uk-width-1-1 uk-flex uk-flex-middle uk-flex-center"
+    :style="{ backgroundImage: 'url(' + backgroundImage + ')' }"
+  >
+    <div class="oc-login-card">
+      <div class="oc-login-card-body">
+        <h3 class="oc-login-card-title">
+          Welcome to ownCloud
```

Comment:
feels a bit generic if it's only the file picker

Reply:
This might be later themeable but for now, I'll put the `ownCloud File picker`